notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-989_analysis_inertia_compensation.ipynb

```diff
@@ -0,0 +1,699 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #35.        def __init__(self, 
</pre><p>Would it make sense to either pass a efd client in the init method or you already create a client on Line 228 that we could store in self. </p><p>I dont really like hardcoding the client outside the class and just referring to it. </p><pre class="ql-syntax" spellcheck="false">Line #228.            events = event_maker.getEvents(day_obs_begin)
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/70/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>